backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -2210,7 +2212,8 @@ void SurfaceSdlGraphicsManager::setMouseCursor(const void *buf, uint w, uint h,
 	}
 
 	if (keycolorChanged) {
-		SDL_SetColorKey(_mouseOrigSurface, SDL_RLEACCEL | SDL_SRCCOLORKEY | SDL_SRCALPHA, _mouseKeyColor);
+		uint32 flags = _disableMouseKeyColor ? 0 : SDL_RLEACCEL | SDL_SRCCOLORKEY | SDL_SRCALPHA;
```

Comment:
Do all of these flags need to be removed? Or just SDL_SRCCOLORKEY?

Reply:
According to https://www.libsdl.org/release/SDL-1.2.15/docs/html/sdlsetcolorkey.html the whole "flags" parameter must be set to 0 to disable the key color, indeed (in SDL1, for SDL2 it is even simpler, the parameter has be changed to be a boolean: https://wiki.libsdl.org/SDL2/SDL_SetColorKey).

Reply:
Thanks